audformat/core/database.py

```diff
@@ -755,6 +791,16 @@ def load_header_from_yaml(header: dict) -> 'Database':
 
         return db
 
+    def _path_absolute(
+        self,
+    ) -> bool:
+        r"""Check if paths in tables are absolute."""
+        return (
```

Comment:
Maybe it's better to simply apply `os.path.isabs()` independent of `self.root`.

Reply:
I switched to `os.path.isabs()` and do not only check the first `files[0]` entry, but now check if any of `files` conatins an absolute path. This is slower, but I think as a user can add whatever paths she likes, there is no way around checking all entries.